10LessonTen/WinerySite/css/style.css

```diff
@@ -0,0 +1,99 @@
+body {
+  font-family: 'Open Sans', sans-serif;
+
+}
+
+nav{
+
+}
+
+#home {
+  background-image: url("../img/wine-bg.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-blend-mode: multiply;
+  background-color: #6e6e6e;
+  color: white;
+  height: 500px;
+  width: 100%;
+  padding-top: 50px;
+  text-align: center;
+}
+
+h2 {
+  font-size: 40px;
+}
+
+#about-us {
+  font-family: sans-serif;
+  height: 500px;
+  width: 100%;
+}
+
+
+.1954 {
+  width: 20%;
+  height: 500px;
+  float: left;
+
+}
+
+.bwimg {
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  float: left;
+  width: 40%;
+  height: 1000px;
+}
+
+
+#testimonials {
+  background-image: url("../img/wine13.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-blend-mode: multiply;
+  background-color: #6e6e6e;
+  font-family: sans-serif;
+  color: white;
+  height: 500px;
+  width: 100%;
+  padding-top: 50px;
+  text-align: center;
+}
+
+
+
+#wine-selection {
+  font-family: sans-serif;
+  padding-top: 50px;
+  text-align: center;
+  margin: auto;
+}
+
+.shiraz {
+  background-image: url("../img/wine7.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  float: right;
+  height: 400px;
+  width: 11%;
+}
+
+.chard {
+  background-image: url("../img/wine3.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  float:right;
+  height: 400px;
+  width: 10%;
+}
+
+.merlot {
+  background-image: url("../img/wine6.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  float: right;
+  height: 400px;
+  width: 10%;
+}
```

Comment:
Can you think of a way to style `.shiraz`, `.chard`, and `.merlot` with the same class, and then just assign that class to each element?